tools/SourceKit/tools/sourcekitd-test/TestOptions.cpp

```diff
@@ -367,6 +367,17 @@ bool TestOptions::parseArgs(llvm::ArrayRef<const char *> Args) {
       return true;
 #endif
 
+    // SWIFT_ENABLE_TENSORFLOW
+    case OPT_in_memory_clang_module_cache:
+#ifdef SWIFT_SOURCEKIT_USE_INPROC_LIBRARY
+      InMemoryClangModuleCache = true;
+      break;
+#else
+      llvm::errs() << "in-memory-clang-module-cache only supported when "
+                      "SWIFT_SOURCEKIT_USE_INPROC_LIBRARY is set";
```

Comment:
Could you elaborate on the reasons behind this restriction?

Reply:
added comment